VideoWeb/VideoWeb/ClientApp/src/app/shared/chat/chat-base.component.ts

```diff
@@ -26,8 +28,11 @@ export abstract class ChatBaseComponent {
         protected eventService: EventsService,
         protected logger: Logger,
         protected adalService: AdalService,
-        protected imHelper: ImHelper
-    ) {}
+        protected imHelper: ImHelper,
+        protected translateService: TranslateService
+    ) {
+        this.name_you = this.translateService.instant('chat-base.you');
```

Comment:
if in ctor it wont update when the language is changed

Reply:
Ohh, see will move

Reply:
well it would but it would require a refresh or component to be reloaded